cppguide.html

```diff
@@ -4374,20 +4377,23 @@ <h4>Class Data Members</h4>
 };
 </pre>
 
+<p><strong>Public</strong> data members of classes do not have the trailing underscore.</p>
+
+<p>In short, the presence of the trailing underscore semantics indicate whether the member variable is public or not.</p>
+
 <h4>Struct Data Members</h4>
 
 <p>Data members of structs, both static and non-static,
-are named like ordinary nonmember variables, <strong>but with a
-trailing underscore.</strong> <del>They do not have
-the trailing underscores that data members in classes have.</del></p>
+are named like ordinary nonmember variables. They do not have
+the trailing underscores that private and protected data members in classes have, since they are all public.</p>
```

Comment:
```suggestion
are named like ordinary nonmember variables, do not have
the trailing underscores that data members in classes have.</del></p>
```

Members of structs are no all public, only if you don't put an access specifier. I would just go for `Data members of structs are named according to the same rule as data members of classes.`

WDYT?